src/main/java/com/github/redouane59/twitter/TwitterClient.java

```diff
@@ -694,8 +724,26 @@ public CollectionsResponse collectionsEntries(final String collectionId, int cou
 
   public static TwitterCredentials getAuthentication() {
     String credentialPath = System.getProperty("twitter.credentials.file.path");
+    if(credentialPath!=null) return getAuthentication(new File(credentialPath));
+    else return getAuthentication(Paths.get(""));
+  }
+  
+  public static TwitterCredentials getAuthentication(final Path pathToScan, final String... validNames) {
```

Comment:
Don't you think that adding few lines on the `README.md` could be useful for future users of the new authentication ways ?

Reply:
From highlevel perspective, there are 2 changes:
1) User can place config file in any directory from current and up
2) User can use a separate config file for tests 
Everything else, as I understand, remains the same